telegram/_files/inputmedia.py

```diff
@@ -83,9 +90,11 @@ def __init__(
         self.type = media_type
         self.media = media
         self.caption = caption
-        self.caption_entities = caption_entities
+        self.caption_entities = tuple(caption_entities) if caption_entities else None
```

Comment:
```suggestion
        self.caption_entities = tuple(caption_entities) if caption_entities else ()
```
might need to modify tests for this in all classes...